content/en/blog/_posts/2024-04-05-diy-create-your-own-cloud-with-kubernetes-part-2/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
+title: "DIY: Create Your Own Cloud with Kubernetes (Part 2)"
+slug: diy-create-your-own-cloud-with-kubernetes-part-2
+date: 2024-04-05
```

Comment:
Let's pick a particular time during this day for the article to publish. They ought to be at least 5 minutes apart, and in order.